From b63edf0ecaab0cb26bba99b139511ac22d3bf785 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Wed, 14 Dec 2011 22:23:56 +0100
Subject: [PATCH 071/126] systemctl: fix typo in 'is-enabled'

It prevented the action from working without dbus.
(cherry picked from commit c971700e41e0ac9883ab0744921b79dd396170dc)
---
 src/systemctl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/systemctl.c b/src/systemctl.c
index 960d3c0..c86ec80 100644
--- a/src/systemctl.c
+++ b/src/systemctl.c
@@ -4998,7 +4998,7 @@ static int systemctl_main(DBusConnection *bus, int argc, char *argv[], DBusError
          * enable/disable */
         if (!streq(verbs[i].verb, "enable") &&
             !streq(verbs[i].verb, "disable") &&
-            !streq(verbs[i].verb, "is-enable") &&
+            !streq(verbs[i].verb, "is-enabled") &&
             !streq(verbs[i].verb, "list-unit-files") &&
             !streq(verbs[i].verb, "reenable") &&
             !streq(verbs[i].verb, "preset") &&
-- 
1.7.7.5

